### ISSUE TITLE
Commented lines without indentation does not get raised as commented code

### ISSUE THREAD

**Issue by sebastiaanspeck** [StevenLooman/sonar-magik#79]
The next snippet should raise multiple issues about commented code, but because "missing" indentation, it does not get all expected issues. Besides that, I see a difference in output when building/testing the linter and when using the linter.

```magik
# testCommentedMethodWithIndentedBlankLine - 2 issues (correct)
#_method a.b
    #_local x << _self.call()
    #x +<< 10
    #x *<< 2
    #
    #write(x)
    #x -<< 5
    #_return x
#_endmethod

# testCommentedMethodWithoutIndentedBlankLine - no issues, in CI 1 issue
#_method a.b
#_local x << _self.call()
#x +<< 10
#x *<< 2
#
#write(x)
#x -<< 5
#_return x
#_endmethod

# testCommentedMethodWithoutBlankLine - 1 issue (correct)
#_method a.b
    #_local x << _self.call()
    #x +<< 10
    #write(x)
    #_return
#_endmethod

# testCommentedMethodWithoutIndentation - 1 issue, in CI no issue
#_method a.b
#_local x << _self.call()
#x +<< 10
#write(x)
#_return x
#_endmethod
```

**Comment by StevenLooman** [comment]
See PR.
